### ISSUE TITLE
Wtype doesn't work with the default typing-key-delay

### ISSUE THREAD

**Issue by Yucklys** [fdw/rofi-rbw#97]
With the resent pull request, Wtype doesn't work with the default typing-key-delay of 0. Here is the output with the default arguments.
```
[<rofi_rbw.models.Target object at 0x7f8ecda4d1d0>, <rofi_rbw.models.TypeTarget object at 0x7f8ecdac9750>, <rofi_rbw.models.Target object at 0x7f8ecda55810>]
Invalid sleep time
Invalid sleep time
```
I guess somewhere in the new code the sleep time is wrong? Currently I need to manually set this arg to 1 to fix.

**Comment by fdw** [comment]
I guess you're not yet on [version 1.4.2](https://github.com/fdw/rofi-rbw/releases/tag/1.4.2), are you?

**Comment by Yucklys** [comment]
> I guess you're not yet on [version 1.4.2](https://github.com/fdw/rofi-rbw/releases/tag/1.4.2), are you?

I was waiting for nixpkgs to update the new version, and it is fixed in 1.4.2. Thanks for your help.

**Comment by fdw** [comment]
You're welcome 🙂
